repos/jsutils/src/node/tryRequire.js

```diff
@@ -0,0 +1,25 @@
+/** @module command */
+
+const fs = require('fs')
+
+/**
+ * Tries to require the path, returning null if unable to.
+ * Does not throw.
+ * @function
+ * @param {string} filePath - path to file. You should use an absolute path
+ * @return {*?} the export at path, if it exists, null otherwise
+ * @example
+ * const module = tryRequire('/keg/tap/foo/bar.js')
+ * if (!module) console.log('bar.js module does not exist')
+ */
+const tryRequire = filePath => {
```

Comment:
Can we make an async version of this as well. It's generally preferred when used cause it's not blocking.

It may not matter for what your doing with tests, but if used by a backend api it could lock things up.

Or if you your code can handle it as async that would best all around

Reply:
Okay I updated this:

`tryRequire` is now asynchronous.

`tryRequireSync` is a new function that is synchronous.

I tried to conform to the node naming scheme here.